### ISSUE TITLE
Sulphur mine unlocks

### ISSUE THREAD

**Issue by keybounce** [doublespeakgames/adarkroom#587]
So I just got the sulphur mine cleared. Much easier with weapons than unarmed.

And: As far as I can tell, the only thing it unlocks is rifle crafting, for 50 sulphur. And that's all it needs.

Bullets are 10 scales, and cannot be crafted.

This seems really odd. A rifle does not need sulfur; a bullet has the powder/primer/explosive in it. The rifle just strikes it and sets it off. (At least, that's my understanding of how it works).

Basically, the sulphur mine seems to not be significant at all. Rifles can be obtained from soldiers easily enough.

**Comment by keybounce** [comment]
Ahh. So eventually the builder mentions an armory and bullets. Just not right away for some reason.

**Comment by keybounce** [comment]
So, I've had the armory built for a while, and NOTHING else to craft.

**Comment by keybounce** [comment]
Ahh. You have to assign people to it in the town. That's ... very NOT expected.

**Comment by AndySky21** [comment]
Basically you're suggesting a notice is introduced in the log to notify the player that a building has been unlocked and therefore a new job is available?

**Comment by gitupofftheflooranddosomework** [comment]
Do we want to add @AndySky21's explanation of the issue?

**Comment by JonnyOThan** [comment]
FWIW I just had this same problem - after building the armory I couldn't figure out how to make bullets until coming here to report the issue.  I had already scavenged some bullets and rifle from the battlefield, so it wasn't obvious that this was something different.
